exonum-java-binding-common/src/main/java/com/exonum/binding/common/configuration/ConsensusConfiguration.java

```diff
@@ -17,49 +17,100 @@
 package com.exonum.binding.common.configuration;
 
 import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
 import com.google.gson.annotations.SerializedName;
-import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
 
 /**
  * Blockchain Consensus algorithm parameters.
```

Comment:
:+1: The amount of details in the docs for each property seems good to me. Would you add please a link to the page where one can get more details at the class level?